_drafts/CocoaPods-1.9.0-beta.markdown

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title:  "CocoaPods 1.9 Beta has arrived!"
+author: amorde
+categories: cocoapods releases
+---
+
+_CocoaPods 1.9_ adds support for .xcframeworks, configuration based dependencies for pod authors, and other ehancements and bug fixes!
+
+<!-- more -->
+
+This release includes several quality-of-life enchancements in addition to important bug fixes.
+
+## XCFramework Support
+
+With the release of Xcode 11, Apple introduced a new bundle format using the `.xcframework` file extension. This format allows multiple copies of a framework compiled for different architectures and platforms to be combined into a single structure.
+
+This release introduces support for Pod authors to ship vendored `.xcframework` bundles using the existing [vendored_framework](https://guides.cocoapods.org/syntax/podspec.html#vendored_frameworks) DSL. Example:
+
+```ruby
+Pod::Spec.new do |s|
+  s.name         = 'ToastLib'
+  s.version      = '1.0.0'
+
+  # ...rest of attributes here
+
+  s.vendored_frameworks = 'ButterLib.xcframework'
+
+end
+```
+
+For apps that include Pods which vendor dynamic frameworks, a new script phase with the name `[CP] Prepare Artifacts` will be added to the project to facilitate including the .xcframework into the app bundle. Be on the look out for a blog post with more details on how this was implemented.
```

Comment:
I think we can remove "Be on the look out for a blog post with more details on how this was implemented." in my opinion. I geeeeenerally avoid public commitments in blog posts just to be on the safe side.

Reply:
yes great point